src/main/webapp/ThemeAndWidgets.xhtml

```diff
@@ -15,12 +15,14 @@
             <ui:param name="dataverse" value="#{themeWidgetFragment.editDv}"/>  
            
             <ui:param name="showMessagePanel" value="#{true}"/>
-            <ui:define name="body">
+            <ui:define name="metadata">
                 <f:metadata>
                     <f:viewParam name="id" value="#{themeWidgetFragment.editDv.id}"/>
                     <f:viewAction action="#{themeWidgetFragment.initEditDv}"/>
                     <f:viewAction action="#{dataverseHeaderFragment.initBreadcrumbs(themeWidgetFragment.editDv, bundle['dataverse.option.themeAndWidgets'])}"/>
                 </f:metadata>
+            </ui:define>
+            <ui:define name="body">
```

Comment:
Should we consider documenting this at a future version of https://guides.dataverse.org/en/latest/developers/tips.html#ui-pages-development ?

Reply:
I see in the PR you wrote this:

"With further testing at QDR, I've noticed a steady/overwhelming stream of errors of the form  `JSF1103: The metadata facet must be a direct child of the view in viewId /dataverse.xhtml]]`.  I did not notice any change in UI behavior though. Digging in a bit, it appears that any page that uses `<ui:composition template="/dataverse_template.xhtml">` was including the f:metadata farther down in the tree. I've made changes to move them to being a direct child of the view which appears to get rid of the log messages, again w/o any impact on functionality that I've seen. I haven't been able to confirm, but my guess is that while the error is now getting flagged, the f:metadata was still be interpreted correctly."

Reply:
Added, thanks!